application/controllers/AuthenticationController.php

```diff
@@ -35,6 +42,36 @@ public function loginAction()
             $this->redirectNow(Url::fromPath('setup'));
         }
         $form = new LoginForm();
+        if (isset($_COOKIE['remember-me'])) {
+            $data = explode('|', $_COOKIE['remember-me']);
+            $publicKey = base64_decode(array_pop($data));
+
+            $select = (new Select())
+                ->from('rememberme')
+                ->columns('*')
+                ->where(['public_key = ?' => $publicKey]);
+
+            $DBData = $this->getDb()->select($select)->fetch();
```

Comment:
Just use `$dbData` if you really want to use that parameter name. I did not go over the "login via remember me cookie" code completely but do that in the next review round ;)